Resources/Locale/ru-RU/_CP14/_PROTO/entities/objects.ftl

```diff
@@ -32,6 +32,17 @@ ent-CP14Crossbolt = арбалетный болт
 ent-CP14WeaponRevolverNavy = револьвер "Марин"
     .desc = Увесистый капсюльный револьвер серии "Марин". В настоящее время револьверы серии "Марин" стоят баснословные деньги, и их могут позволить себе исключительно преуспевающие моряки и торговцы.
 
+ent-CP14CartridgeBulletRifle = винтовочный патрон
```

Comment:
things here are sorted by category as they are arranged in folders. Since ammunition is separate from weapons, they need a separate category here

Reply:
done